opm/simulators/wells/WellInterfaceGeneric.cpp

```diff
@@ -487,6 +487,13 @@ WellInterfaceGeneric<Scalar>::getDynamicThpLimit() const
     return dynamic_thp_limit_;
 }
 
+template<class Scalar>
+void WellInterfaceGeneric<Scalar>::
+setDynamicThpLimit(const std::optional<Scalar> thp_limit)
```

Comment:
This function is node needed, please remove.

Reply:
Removed it.